noir-projects/noir-protocol-circuits/crates/private-kernel-lib/src/tests/validate_arrays.nr

```diff
@@ -32,20 +35,19 @@ fn validate_arrays_malformed_nullifier_read_requests_fails() {
     builder.validate();
 }
 
-// Enable this test if MAX_NULLIFIER_KEY_VALIDATION_REQUESTS_PER_CALL is greater than 1.
```

Comment:
Re-enabled this test due to this comment ^

I didn't modify the constant (just the name of it) so it should have been done in the PR which increased it.